src/renderer/components/+events/events.tsx

```diff
@@ -163,7 +163,7 @@ export class Events extends React.Component<Props> {
           { title: "Involved Object", className: "object", sortBy: columnId.object, id: columnId.object },
           { title: "Source", className: "source", id: columnId.source },
           { title: "Count", className: "count", sortBy: columnId.count, id: columnId.count },
-          { title: "Last Seen", className: "age", sortBy: columnId.age, id: columnId.age },
+          { title: "Age", className: "age", sortBy: columnId.age, id: columnId.age },
```

Comment:
I think we should fix "Last Seen" instead of this.

Reply:
How about both?

Reply:
what is meant by "Last Seen"? It can mean "when an event was last seen", which would be the age of the event. Or it can mean "show me the last event seen", so the default order would be from most recent to oldest(?) Or does it mean when a specific event was last seen, which would be an actual time value. I think "Age" makes it unambiguous.

Reply:
@jim-docker See discussion on https://github.com/lensapp/lens/issues/2819. Some controllers increment the count and last seen time instead of creating a new one.